scripting/togglemusic_dhook.sp

```diff
@@ -1,17 +1,11 @@
-#pragma semicolon 1
-
-#include <sourcemod>
```

Comment:
why are you removing the basic sm include?

Reply:
SM dosent needs to be included in order for the Plugin to work, it compiles fine on my Side.